java/flight/flight-sql-jdbc-core/src/main/java/org/apache/arrow/driver/jdbc/client/ArrowFlightSqlClientHandler.java

```diff
@@ -463,8 +466,37 @@ public Builder withSystemTrustStore(final boolean useSystemTrustStore) {
       return this;
     }
 
+
+    public Builder withTlsRootCertificates(final String tlsRootCertificatesPath) {
```

Comment:
nit: shall we add documentation?

Reply:
hi, @vibhatha - Noted - corrected and pushed a change just now.